### DISCUSSION TITLE
No connection to inverter openDTU

### DISCUSSION THREAD

**Discussion by roborol** [tbnobody/OpenDTU#1205] [General]
Hello, maybe someone has an idea. I cannot connect to the inverter (HMT-2250-6T).
Thanks a lot !
![2023-08-04 08_56_36-OpenDTU – Mozilla Firefox](https://github.com/tbnobody/OpenDTU/assets/138994213/cf714d5a-ff58-47d0-8df0-80494c01b1ed)
![2023-08-04 09_00_09-OpenDTU – Mozilla Firefox](https://github.com/tbnobody/OpenDTU/assets/138994213/7ee3e256-3dba-4cbb-949b-51c972424f8b)
![2023-08-04 09_01_58-OpenDTU – Mozilla Firefox](https://github.com/tbnobody/OpenDTU/assets/138994213/e0fcd45f-302a-4890-b5f5-2a4f0f23986f)

**Comment by tbnobody** [comment]
Du solltest ggf. die Konsole ein paar Sekunden länger offen lassen bevor du einen Screenshot machst. Mit nur einer Zeile lässt sich hier leider nicht viel sagen.

**Reply by roborol** [reply]
Vielen Dank für deine Rückmeldung !
Mehr kommt da nicht in der Konsole

**Comment by tbnobody** [comment]
Dann hast du entweder keinen inverter eingetragen oder er ist mit einer unbekannten nummer eingetragen. wie schaut denn deine Liste unter Einstellungen --> Inverter aus?

**Reply by roborol** [reply]
Jetzt habe ich leider keinen Zugriff mehr, gibt es eine Möglichkeit das Teil zurück zu setzen ?
![2023-08-04 18_24_05-](https://github.com/tbnobody/OpenDTU/assets/138994213/408d8a0c-4d9d-4263-a4a3-17c1cd97678e)

**Comment by tbnobody** [comment]
Wenn du das passwort geändert hast (für die WebUI) ist das wlan passwort genau dieses Passwort für die Web Oberfläche.

But if you've already placed the DTU into your local WiFi the external accesspoint will close. And you have to place it in your local Wifi otherwise it will not get a time from a timeserver (which is required to send requests to the inverter). This is also mentioned in the readme and wiki

**Reply by roborol** [reply]
Hab kein PW geändert nur die IP (Netzwerk) per Hand eingetragen (ohne DHCP), jetzt habe ich weder über diese IP (mein Hausnetz) noch über die OpenDTU-13243616 Zugriff :-(

**Reply by DPO99** [reply]
Ist die OpenDTU nun mit deinem Netzwerk verbunden oder nicht? Entweder das Ding ist im Netz und dann über die IP Aufrufbar oder du solltest über den AP reinkommen.

Und Du bist dir sicher, dass die IP Daten (IP & Netzmaske & Standardgateway) korrekt eingetragen hast? Wieso ist dhcp eigentlich aus?

**Reply by roborol** [reply]
Nein ich komme leider nicht mehr über die eine noch die andere Möglichkeit an die DTU ran.  IP 10.0.0.18 / Subnet 255.255.2550 / Standag, DNS 1,2 10.0.0.138.

**Reply by DPO99** [reply]
Naja dann neu flashen, dann geht der AP wieder an.

**Reply by roborol** [reply]
ok !

**Reply by roborol** [reply]
Über AP komme bin ich drauf hab den DHCP wieder aktiviert. Über die IP (10.0.0.18) zu meinem Hauswerk komme ich nicht (anpingen funktioniert), ich denke schön langsam das Teil was mir da verkauft wurde ist defekt, hab auch alle Lötstellen 2 mal mit Lupe kontrolliert.
![2023-08-06 09_04_38-OpenDTU – Mozilla Firefox](https://github.com/tbnobody/OpenDTU/assets/138994213/da79fcc8-80f7-406c-9c38-528da9b648b4)

**Reply by joergw69** [reply]
Wenn ich das richtig sehe, versuchst Du einen HMT mit dem NFR Chip zu erreichen. Für den HMT und HMS brauchst Du aber das CMT2300A
